test/function/samples/internal-reexports-from-external/_config.js

```diff
@@ -1,16 +1,13 @@
-const path = require('path');
+const assert = require('assert');
+const fs = require('fs');
 
 module.exports = {
 	description:
 		'fails with a helpful error if creating a namespace object containing a reexported external namespace',
```

Comment:
Nice test comparing with fs! The test description is no longer valid, though. Also, it would be nice to add a `form` test to inspect the generated output.

Reply:
I've fixed up the description and added a form test.